src/Database/Query.php

```diff
@@ -710,7 +710,7 @@ protected function execute($sql, array $params = [])
             $this->database->fail();
         }
 
-        $result = $this->database->execute($sql['query'], $sql['bindings'], $params);
```

Comment:
Why is this being removed?

Reply:
Because `execute()` method accepts only two arguments, am i wrong about called `execute()` method?

https://github.com/getkirby/kirby/blob/develop/src/Database/Database.php#L467

Reply:
You are right again!